tests/ui/cloned_last.rs

```diff
@@ -7,4 +7,16 @@ fn main() {
         .iter()
         .cloned()
         .last();
+
+    #[rustfmt::skip]
```

Comment:
Why skip rustfmt?

Reply:
I copied it from another example. It's not really needed, so I removed it